dvc/stage.py

```diff
@@ -451,7 +451,7 @@ def create(repo, **kwargs):
             if fname is not None and os.path.basename(fname) != fname:
                 raise StageFileBadNameError(
                     "DVC-file name '{fname}' may not contain subdirectories"
-                    " if '-c|--cwd' (deprecated) is specified. Use '-w|--wdir'"
+                    " if `-c|--cwd` (deprecated) is specified. Use '-w|--wdir'"
```

Comment:
Consistency with what? You have '-w|--wdir' literally in the same line.

Reply:
Oops. For some reason I only found this one instance in all the docs the first time I checked for commands or arguments in `'` inside `"` strings... There's lots more! Updated in 9e2b71c4...